app/models/rules/active_in_cohort.rb

```diff
@@ -19,19 +20,24 @@ def available_cohorts
   end
 
   def display_for_variable value
-    available_cohorts.to_h.try(:[], value.to_i) || value
+    # Note, we need to jump through a few hoops since variable wasn't designed as a json field
+    available_cohorts.select { |id, _| id.to_s.in?(value_as_array(value)) }.map(&:last).join(' or ') || value
   end
 
-  def clients_that_fit(scope, requirement, opportunity)
+  def clients_that_fit(scope, requirement, _opportunity)
     if Client.column_names.include?(:active_cohort_ids.to_s)
       if requirement.positive
-        where = 'active_cohort_ids @> ?'
+        where = 'active_cohort_ids @> ANY(ARRAY [?]::jsonb[])'
```

Comment:
I'm not familiar with this syntax. Can you tell me what it's doing?

Reply:
The question mark will be replaced with the value when dropped into the where statement down below.  They @> is postgres json query syntax for "in" (https://www.postgresql.org/docs/9.5/functions-json.html.)  So, force the string provided into an postgres array, and see if any of the provided values are in the active_cohort_ids column.

Reply:
It was the `@>` that was new to me. Thanks!